src/components/AdjMatrix/AdjMatrixMethods.ts

```diff
@@ -531,7 +539,7 @@ export class View {
       .attr('x', verticalOffset)
       .attr('dy', '.32em')
       .attr('text-anchor', 'start')
-      .style('font-size', this.nodeFontSize)
+      .style('font-size', this.nodeFontSize.toString() + 'pt')
```

Comment:
This is already a string so we can just do (there are 2 places to change this):
```suggestion
      .style('font-size', `${this.nodeFontSize}pt`)
```